### ISSUE TITLE
UI: Good contrast / WCAG compatibility

### ISSUE THREAD

**Issue by 6543** [woodpecker-ci/woodpecker#1315]
" ... WCAG compatibility is pretty important, at least AA grade ..."

----
https://github.com/woodpecker-ci/woodpecker/issues/468#issuecomment-1075089963, https://github.com/woodpecker-ci/woodpecker/issues/468#issuecomment-1207138477

**Comment by 6543** [comment]
@tecosaur:

This seems like the place to plop UI feedback? I've just started using Woodpecker and I'm sorry to say that the dark mode contrast is atrocious.

![image](https://user-images.githubusercontent.com/20903656/183231941-f30595e9-0604-4ca8-9530-395830021366.png)

Let alone WCAG AAA, this fails WCAG AA standards of accessibility.

![image](https://user-images.githubusercontent.com/20903656/183232110-aab14b51-f757-4176-947c-9686d80e4e8b.png)
![image](https://user-images.githubusercontent.com/20903656/183232124-3ef03ce3-b9b5-40ee-a78c-30a9da5e8d78.png)

I recommend https://accessible-colors.com/ for checking that there's sufficient contrast between foreground and background colours.

Here's what `#CCCFD4` looks like:

![image](https://user-images.githubusercontent.com/20903656/183232170-a6322f94-ce30-4110-9922-b87603b25f21.png)

Copying Gitea's default foreground `#bbc0ca` is probably another decent option (it's 2/3 of the way to AAA from AA).

On a more stylistic note, I don't think the dark borders do the design any favours either. Here's a lighter version:

![image](https://user-images.githubusercontent.com/20903656/183232207-87b3e3ab-bac7-42ec-9ca2-3777f825b268.png)

There are other improvements that can be made, but this would be a good starting point I think :slightly_smiling_face:.

---
https://github.com/woodpecker-ci/woodpecker/issues/468#issuecomment-1207138477

**Comment by 6543** [comment]
https://wave.webaim.org/
